inference/wasm/tests/test-cases/translate-html-multi-paragraph-no-pivot.test.mjs

```diff
@@ -55,6 +55,15 @@ const testCases = [
       "<b>The blue dog. </b><u><i>It runs in the park.\n\nIt looks very happy. </i></u>It's playing with other dogs.",
     isHTML: true,
   },
+  {
+    sourceLanguage: "en",
+    targetLanguage: "zh",
```

Comment:
Suggestion: This may be surprising for future code wanderers without a comment explaining it. Squash merges remove these nice commit messages, yay GitHub. Maybe just do a quick search/replace for `targetLanguage: "zh",` to add a comment.

Reply:
Agreed, I'll add comments to all of them.